### PR TITLE
SK-1614-WEB-Domain-DNS-DKIM-v2

### PR DESCRIPTION
**Pull request by BrenKt**

### CODE REVIEW

**Comment by BrenKt** [comment]
New update to clarify selectors status

File: pages/web_cloud/domains/dns_zone_dkim/guide.fr-fr.md [hunk 1]
```diff
@@ -370,6 +370,40 @@ Suivez les **5 étapes** en cliquant successivement sur chacun des 5 onglets ci-
 >>
 > **5. Activation du DKIM**
 >>
+>> Après propagation de votre configuration DNS, utilisez à nouveau l'appel API suivant pour activer le DKIM :<br>
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dkim/guide.fr-fr.md]:
```suggestion
>> Après la propagation de votre configuration DNS, utilisez à nouveau l'appel API suivant pour activer le DKIM :<br>
```

File: pages/web_cloud/domains/dns_zone_dkim/guide.fr-fr.md [hunk 2]
```diff
@@ -370,6 +370,40 @@ Suivez les **5 étapes** en cliquant successivement sur chacun des 5 onglets ci-
...
+>>
+>> > [!api]
+>> >
+>> > @api {v1} /email/domaine/ PUT /email/domaine/{domain}/dkim/enable
+>>
+>> - `domain` : saisissez le nom de domaine attaché à votre service e-mail sur lequel vous souhaitez activer DKIM.
+>>
+>> Cliquez sur `TRY`{.action} pour lancer l'activation.<br>
+>>
+>> *Exemple de résultat:*
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dkim/guide.fr-fr.md]:
```suggestion
>> *Exemple de résultat :*
```

File: pages/web_cloud/domains/dns_zone_dkim/guide.fr-fr.md [hunk 3]
```diff
@@ -709,14 +743,20 @@ Sélectionnez l'offre e-mail concernée dans les onglets suivant:
 >>
 >> - `domain` : saisissez le nom de domaine attaché à votre service E-mail sur lequel le DKIM doit être présent.
 >>
->> Regardez ensuite la valeur `status:` dans le résultat :
+>> Regardez ensuite la valeur `status:` général dans le résultat :
```
Comment (OVH-Team-Guides) [inline pages/web_cloud/domains/dns_zone_dkim/guide.fr-fr.md]:
```suggestion
>> Regardez ensuite la valeur `status:` générale dans le résultat :
```

**Pull request merged**